enterprise/backend/src/metabase_enterprise/enhancements/models/native_query_snippet/permissions.clj

```diff
@@ -18,7 +18,7 @@
 
 (defenterprise can-read?
   "Can the current User read this `snippet`?"
-  :feature :any
+  :feature :advanced-permissions
```

Comment:
looks like we're still discussing which flag we should use for this https://www.notion.so/Feature-Flag-Verification-Table-669da06350984fde8d18011b8f8f0b14?d=e321e65df39b4901b4b8e6cdb1542e00&pvs=4#0f1de05885bd4a02a9e08343f82c6ecf